### ISSUE TITLE
Change test to validation, add 3-way dataset split

### ISSUE THREAD

**Issue by nickgreenquist** [nickgreenquist/cu2rec#51]
right now, we are using our test set as a validation set (ie basing learning rate of it for example). We should change our split function to return a 70/10/10 split.
